docker-compose.yml

```diff
@@ -1,91 +1,85 @@
 version: "3.9"
 services:
   mysql:
-    image: mysql:8.0.27
-    command: --default-authentication-plugin=mysql_native_password
+    image: mysql:5.7
     restart: always
     environment:
       MYSQL_ROOT_PASSWORD: mysql_root_password
-      MYSQL_USER: root
       MYSQL_DATABASE: app
-    ports:
-      - "33061:3306"
+    expose:
+      - "3306"
     volumes:
       - mysql:/var/lib/mysql
-    logging:
-      driver: none
   rabbitmq:
     image: rabbitmq:3.9.12
     restart: always
     environment:
       RABBITMQ_NODENAME: rabbit
-    ports:
-      - "5672:5672"
-      - "15672:15672"
-      - "25672:25672"
-      - "4369:4369"
+    expose:
+      - "5672"
+      - "15672"
+      - "25672"
+      - "4369"
   memcached:
     image: memcached:1.6.12
     restart: always
-    ports:
-      - "11211:11211"
+    expose:
+      - "11211"
   mongodb:
     image: mongo:5.0.4
     restart: always
     volumes:
       - /data/db
-    ports:
-      - "27017:27017"
+    expose:
+      - "27017"
   postgres:
-    image: postgres:14.1
+    image: postgres:14.2
```

Comment:
What are your thoughts about pulling directly from `postgres:latest` instead?

Reply:
My take: we should only commit the specific tags that we've personally tested with and acknowledged as working. I'd be more inclined to switch to "latest" if the latest version was actively being brought down and tested by an automated process (CI for PRs or whatnot) and we could at least have some proactive heads up when a breaking change is introduced.

I'm very happy to consider benefits to going with "latest" though, let me know.

As far as this bump here from 14.1 -> 14.2, it wasn't actually necessary in the end. The issue I was seeing turned out to be due to the `database.yml` changes needed in this PR. But after doing so much testing on 14.2 I went ahead and left it.

Reply:
I completely agree. The main benefits I considered were avoiding minor bump image updates since it seemed like the latest version was regularly being adopted. I agree that we should be testing these things before the versions change, especially because people outside our organization may be using this tool and I'd hate to have it break on them.